.changeset/chilly-badgers-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/brand-primitives': patch
+---
+
+Fixes duplicate typographic scale sizes on small viewports
```

Comment:
Can we remove the changeset on this occasion please. The global typography change hasn't been released yet, so this fix can piggyback off that existing one in `main`.